scripts/add-helm-code.sh

```diff
@@ -72,6 +72,7 @@ if [[ $CMD == "add" ]]; then
   done < "$op_file"
 
   # Loop over all yaml files in the directory
+  "${HOME_DIR}/hack/check_pre_req.sh" --assert yq
```

Comment:
Since there is no `set -e` in this script, a failed prereq check here will not stop the script from blundering on. Don't we want `set -e` in this script?

Reply:
ok, done